event/event.go

```diff
@@ -1,7 +1,17 @@
 package event
 
-// TODO: remove hello called example model
-// HelloCalled provides an avro structure for a Hello Called event
-type HelloCalled struct {
-	RecipientName string `avro:"recipient_name"`
+import (
+	"github.com/ONSdigital/dp-import/events"
+)
+
+// InstanceStarted provides an avro structure for a Instance Started event
+type InstanceStarted events.CantabularDatasetInstanceStarted
```

Comment:
I think we don't need this alias, but it reduced the package dependencies for other packages in the service. I think it makes sense to leave it :)

Reply:
I mainly did this for readability purposes, so when imported it can read like `event.InstanceStarted` rather than 'events.CantbularDatasetInstanceStarted`, but yeah I guess it has this benefit too :)